state/state.yml

```diff
@@ -52,3 +52,5 @@ properties:
         patternProperties:
           "^(eth|lan|wan)[0-9]*$":
             $ref: "https://ucentral.io/state/v1/link-state/"
+  mac-address-list:
+    $ref: './mac-address-list.yml'
```

Comment:
This is not properly parsed by the merge-schema.py script, even if you fix the errors in the mac-address-list.yml file itself, you would get the following:
        "mac-address-list": {
            "properties": {
                "reference": {
                    "type": "string"
                }
            }
        }